src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,10 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        if ((email.equals("bob@i.ua") && password.equals("1234"))
+                || (email.equals("alice@i.ua") && password.equals("1234"))) {
+            return true;
+        }
```

Comment:
Please, consider refactoring `login` implementation in the way it will use `findByEmail` for user fetching.